src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -12,19 +12,20 @@
             cursor: pointer;
         }
 
-        .signpost-action {
-            min-width: $clr_baselineRem_1;
-            margin: 0;
-            padding: 0;
+        .signpost-trigger {
             color: $clr-dark-midtone-gray;
-            outline: none;
+            border: none;
```

Comment:
hmm we need to remove this border styles too. If users want to use their own custom trigger element and if that element has a border, we are removing that here.

Reply:
ok. I'll remove it.